libs/runtime/ebpf_hash_table.h

```diff
@@ -215,6 +214,27 @@ extern "C"
     size_t
     ebpf_hash_table_key_count(_In_ const ebpf_hash_table_t* hash_table);
 
+    /**
+     * @brief Returns the next (key, value) pair in the hash table that is lexicographically after the specified key
+     * using the provided comparison function and filter function.
+     *
+     * @param[in] hash_table Hash-table to query.
+     * @param[in] previous_key Previous key or NULL to restart.
+     * @param[out] next_key_pointer Pointer to next key if one exists.
+     * @param[out] next_value If non-NULL, returns the next value if it exists.
+     * @retval EBPF_SUCCESS The operation was successful.
+     * @retval EBPF_NO_MORE_KEYS No keys exist in the hash table.
```

Comment:
This message seems wrong.  i think it should match the message that used to be in lines 199-200

Reply:
Let me know if the description is better now.